{"name": "Проверка невозможности логина с неверным паролем", "status": "passed", "start": 1728004404368, "stop": 1728004405099, "uuid": "eecf9564-f0a2-4d0f-a95f-ec0ce88aafd7", "historyId": "f172cd31ecd683675313372bc5f1ea3a", "testCaseId": "f172cd31ecd683675313372bc5f1ea3a", "fullName": "tests.test_login_user.TestsLoginUser#test_not_login_with_invalid_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestsLoginUser"}, {"name": "host", "value": "DESKTOP-LJDD8TD"}, {"name": "thread", "value": "5224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}